src/GrpcProtos/capture.proto

```diff
@@ -452,6 +459,8 @@ message MetadataEvent {
 }
 
 message ClientCaptureEvent {
+  reserved 23, 28, 29;
```

Comment:
@dimitry- What happens with backwards compatibility here?

Reply:
these events are going to be ignored when loading from old files.

Reply:
also see my commend for the case in the client

Reply:
Thanks for the suggestion but I cannot find the comments for the case in the client 😂